docs/content/docs/dev/datastream/fault-tolerance/checkpointing.md

```diff
@@ -292,4 +292,25 @@ The final checkpoint would be triggered immediately after all operators have rea
 without waiting for periodic triggering, but the job will need to wait for this final checkpoint 
 to be completed.
 
+## Unify file merging mechanism for checkpoints
+
+Flink 1.20 introduces a unified file merging mechanism for checkpointing, which allows scattered small checkpoint files 
```

Comment:
Is it a mvp or production ready in Flink 1.20 ?
We'd better to describe it if it's a mvp.